packages/fetch-and-compile/test/fetch.test.ts

```diff
@@ -0,0 +1,118 @@
+import assert from "assert";
+import "mocha";
```

Comment:
I pointed this out elsewhere but given how big the conversation on this PR has gotten you might have missed it, so, I wanted to point out again that this line can be deleted.  Rather than explain in more detail here I'll just link to [where I talked about this before](https://github.com/trufflesuite/truffle/pull/4424#discussion_r783526144).

Reply:
Hmmmm, so I tried removing this line and the tests fail without it. I modeled `fetch.test.ts` after these these tests that @eggplantzzz shared with me https://github.com/trufflesuite/truffle/blob/develop/packages/hdwallet-provider/test/urlValidation.test.ts and they seem to import `assert` and `{ describe, it }`. Could this be because they're `ts` and not `js`? Also the error I'm getting are `TSError: ⨯ Unable to compile TypeScript:
test/fetch.test.ts:24:1 - error TS2304: Cannot find name 'afterEach'` in case it helps.

Reply:
Oh, weird!  Yeah I guess I don't know much of anything about using mocha with TS.  I guess maybe I'm just wrong about this one!